project_timesheet_holidays_contract/tests/common.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from odoo import Command
+from odoo.tests import common
+
+
+class ProjectTimesheetHolidaysContractCommon(common.TransactionCase):
```

Comment:
```suggestion
class ProjectTimesheetHolidaysContractCommon(common.BaseCase):
```
To have all tracking disabled

Reply:
To have all tracking disabled:
````
from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT


class ProjectTimesheetHolidaysContractCommon(common.TransactionCase):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
````